### DISCUSSION TITLE
Planet calculations

### DISCUSSION THREAD

**Discussion by GTFreeFlyer** [Stellarium/stellarium#2599] [Q&A]
Hi, I'm trying to create a GUI that will work with Stellarium.  I've done all my planet calculations using the Keplerian orbital elements.  It works really well, but my accuracy is just a bit off.  The planets appear in the eyepiece, but towards the edge, about .1 to .3 degrees off-center after doing a go-to with my CGX-L mount.  When I use Stellarium or CPWI, the planet appears almost dead-center in the eyepiece.

I have scoured the internet looking for the most up to date orbital elements (and the rate at which they change), and I just cannot get my solution to be as accurate as Stellarium or CPWI.  Where is this information and calculation stored in the repository?  I want to double-check my calcs.

Thanks so much in advance!

**Comment by gzotti** [comment]
First, make sure to compare coordinates in the correct coordinate system, e.g. J2000 vs. Jnow. This alone may already amount to almost 1/3 degree. Then, Keplerian elements change, so you must use osculating elements valid for your date. 
The Stellarium User Guide contains details about Stellarium's sources of accuracy.
